deployment/README.md

```diff
@@ -0,0 +1,6 @@
+This dir contains terraform information for the AWS resources that are needed for the ghc.dev hosting.
+The state is stored in an aws bucket on our account.
+
```

Comment:
Mention the need for an API TOKEN and to set `AWS_PROFILE` environment variable.